about.html

```diff
@@ -91,25 +91,31 @@
                       </div>
                       <!-- HAMBURGER WRAP -->
                       <div class="container responsivegrid hamburger">
                         <div id="container-686bb06b29" class="cmp-container">
                           <!-- HAM OPEN -->
-                          <div class="image open">
-                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container/container_2037006891/container/container/image_324769233.coreimg{.width}.svg/1669059161137/hamburger.svg" data-asset="/content/dam/LeaderImpact/site-./assets/hamburger.svg" data-title="menu" id="image-479635272c" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
-                              <img src="./assets/hamburger.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="menu">
-                            </div>
+                          <div class="button image open">
+                            <button type="button" id="button-5487aab983" class="cmp-button" data-cmp-clickable="" data-cmp-data-layer="{&quot;button-5487aab983&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/button&quot;,&quot;dc:title&quot;:&quot;Button&quot;}}">
+                              <span class="cmp-button__icon cmp-button__icon--open open" aria-hidden="true">
+                                <img src="./assets/hamburger.svg" class="cmp-image-button__icon" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
+                              </span>
+                            </button>
                           </div>
                           <!-- HAM CLOSE  -->
-                          <div class="image close">
-                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container/container_2037006891/container/container/image.coreimg{.width}.svg/1669059409586/close.svg" data-asset="/content/dam/LeaderImpact/site-./assets/close.svg" data-title="Close" id="image-534f826118" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
-                              <img src="./assets/close-menu.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
-                            </div>
+                          <div class="button image close">
+                            <button type="button" id="button-5487aab983" class="cmp-button" data-cmp-clickable="" data-cmp-data-layer="{&quot;button-5487aab983&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/button&quot;,&quot;dc:title&quot;:&quot;Button&quot;}}">
+                              <span class="cmp-button__icon cmp-button__icon--close close" aria-hidden="true">
+                                <img src="./assets/close-menu.svg" class="cmp-image-button__icon" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
```

Comment:
@Prystal254 Button components don't use `<img />` for the icons. You need to use a pseudo element.